packages/angular_devkit/build_angular/src/babel/plugins/adjust-angular-classes_spec.ts

```diff
@@ -0,0 +1,555 @@
+/**
+ * @license
+ * Copyright Google LLC All Rights Reserved.
+ *
+ * Use of this source code is governed by an MIT-style license that can be
+ * found in the LICENSE file at https://angular.io/license
+ */
+
+import { transform } from '@babel/core';
+import { default as adjustAngularClasses } from './adjust-angular-classes';
```

Comment:
I find the name here slightly misleading as this does more than just modify "Angular" classes. It visits all TypeScript classes and amended static properties, and wraps decorators.

Reply:
Indeed.  the plugin started out as Angular only but since the logic was nearly identical for the non-Angular cases they were combined.  I'll change the name to account for this.